benchmarks/transactions-generator/src/account.rs

```diff
@@ -0,0 +1,71 @@
+use anyhow::Context;
+use near_crypto::{InMemorySigner, PublicKey, SecretKey, Signer};
+use near_primitives::types::AccountId;
+use serde::{Deserialize, Serialize};
+use std::fs;
+use std::path::Path;
+
+#[derive(Serialize, Deserialize, Clone, Debug)]
+pub struct Account {
+    #[serde(rename = "account_id")]
+    pub id: AccountId,
+    pub public_key: PublicKey,
+    pub secret_key: SecretKey,
+    // New transaction must have a nonce bigger than this.
+    pub nonce: u64,
+}
+
+impl Account {
+    pub fn new(id: AccountId, secret_key: SecretKey, nonce: u64) -> Self {
+        Self { id, public_key: secret_key.public_key(), secret_key, nonce }
+    }
+
+    pub fn from_file(path: &Path) -> anyhow::Result<Account> {
```

Comment:
Can this function not just panic?  If it is failing then that means we will probably fail to start the application.  I imagine that there isn't much that can be done [from within the application] if this fails.  I am not a fan of unnecessary error passing and handling in relatively straightforward cli applications.

Reply:
further would extend this comment to the rest of the PR.

Also please do not invest time making changes to address this review yet till we have alignment on the overall approach.